### DISCUSSION TITLE
Next.JS for SSR

### DISCUSSION THREAD

**Discussion by SumitMunot** [NeoSOFT-Technologies/frontend-reactjs#8] [Ideas]
We need to have SSR to be considered for web app SEO point of view

**Comment by santoshshinde2012** [comment]
1. SEO 
2. Security 
3. lightweight 
4. SSR
5. Open Graph protocol

**Reply by santoshshinde2012** [reply]
Server-side rendering (SSR), is the ability of an application to contribute by displaying the web page on the server instead of rendering it in the browser. Server-side sends a fully rendered page to the client; the client's JavaScript bundle takes over and allows the SPA framework to operate.

**Comment by santoshshinde2012** [comment]
@SumitMunot  What do you think about this boilerplate?

https://github.com/pankod/next-boilerplate
